Sources/Apodini/Information/InformationClass.swift

```diff
@@ -0,0 +1,54 @@
+//
+// Created by Andreas Bauer on 06.07.21.
+//
+
+/// A ```InformationClass``` is a way to group multiple, similar ``Information`` types
+/// which represent the same Metadata information and can be mapped to the same type of
+/// key and value.
+/// A ``InformationClass`` is typically defined for a specific ``InterfaceExporter`` or a group
+/// ``InterfaceExporter`` sharing the same middleware.
+/// An ``InterfaceExporter`` uses an predefined ``InformationClass`` to collect all ``Information`` instances
+/// which shall be mapped to a certain wire format feature (in both direction). HTTP Headers are one example for that.
+///
+/// Below examples consider the example of a `AnyExampleInformation` (conforming to ``Information``) which is
+/// identified by a `String` and has a value of type `String`.
+///
+/// ## Defining a new ``InformationClass``
+/// While ``InformationClass`` protocol doesn't explicitly define any requirements (to avoid associated types requirements constraints),
+/// all implementations should follow the same style to be consistent.
+///
+/// ```swift
+/// public protocol ExampleInformationClass: InformationClass {
+///     var entry: (key: String, value: String) { get }
+/// }
+/// ```
+///
+/// The `entry` tuple matches the key and value types.
+///
+/// ## Declaring a ``Information`` to be part of a ``InformationClass``
+/// Now we want to declare the `AnyExampleInformation` to be part of the above-defined `ExampleInformationClass`.
+///
+/// ```swift
+/// // declaring protocol conformance
+/// extension AnyExampleInformation: ExampleInformationClass {}
+///
+/// // We declare the actual implementation of the class as an extension on the `InformationClass` itself.
```

Comment:
Double ticks

Reply:
This represents a comment inside `swift` code blocks. So single ticks are fine right?